ci/build/build-code-server.sh

```diff
@@ -32,14 +31,10 @@ main() {
     set -e
   fi
 
-  parcel build \
-    --public-url "." \
-    --dist-dir dist \
-    $([[ $MINIFY ]] || echo --no-optimize) \
-    src/browser/register.ts \
-    src/browser/serviceWorker.ts \
-    src/browser/pages/login.ts \
-    src/browser/pages/vscode.ts
+  yarn browserify src/browser/register.ts -o out/browser/register.browserified.js
+  yarn browserify src/browser/serviceWorker.ts -o out/browser/serviceWorker.browserified.js
```

Comment:
We don't use any imports in the service worker so we could use it as-is!

Reply:
Oh fantastic! Then will remove that here and use the non-browserified `.js` file in the HTML file. Nice catch!